pkg/udf/reduce_udf.go

```diff
@@ -140,10 +140,10 @@ func (u *ReduceUDFProcessor) Start(ctx context.Context) error {
 	var pbqManager *pbq.Manager
 	// TODO: revisit to see if we need to use more storeOptions
 	if storage := u.VertexInstance.Vertex.Spec.UDF.GroupBy.Storage; storage != nil && storage.PersistentVolumeClaim != nil {
-		pbqManager, err = pbq.NewManager(ctx, pbq.WithPBQStoreOptions(store.WithPbqStoreType(dfv1.FileSystemType)))
+		pbqManager, err = pbq.NewManager(ctx, pbq.WithPBQStoreOptions(store.WithPbqStoreType(dfv1.FileSystemType), store.WithMaxBufferSize(dfv1.DefaultStoreMaxBufferSize), store.WithSyncDuration(dfv1.DefaultStoreSyncDuration)))
```

Comment:
Can you rebase on top of the recent store provider change? Also, if it is default then better do it inside the specific store provider in this case for `FileSystemType`.